example_rails6/app/javascript/stylesheets/variables.scss

```diff
@@ -0,0 +1,312 @@
+$color-primary-600: hsl(208, 80%, 52%);
+$color-primary-700: hsl(208, 77%, 47%);
+$color-primary-800: hsl(209, 77%, 43%);
+$color-primary-900: hsl(209, 75%, 38%);
+
+$color-secondary-50: hsl(51, 99%, 93%);
+$color-secondary-100: hsl(48, 100%, 88%);
+$color-secondary-200: hsl(49, 99%, 80%);
+$color-secondary-300: hsl(48, 100%, 70%);
+$color-secondary-400: hsl(47, 100%, 61%);
+$color-secondary-500: hsl(45, 97%, 54%);
+$color-secondary-600: hsl(42, 96%, 50%);
+$color-secondary-700: hsl(39, 100%, 48%);
+$color-secondary-800: hsl(35, 99%, 47%);
+$color-secondary-900: hsl(31, 99%, 45%);
+
+$color-neutral-50: hsl(207, 19%, 98%);
+$color-neutral-100: hsl(208, 19%, 95%);
+$color-neutral-200: hsl(214, 15%, 93%);
+$color-neutral-300: hsl(213, 14%, 89%);
+$color-neutral-400: hsl(208, 13%, 83%);
+$color-neutral-500: hsl(208, 11%, 71%);
+$color-neutral-600: hsl(208, 7%, 56%);
+$color-neutral-700: hsl(208, 9%, 31%);
+$color-neutral-800: hsl(208, 10%, 23%);
+$color-neutral-900: hsl(213, 11%, 15%);
+
+$color-white: #ffffff;
+$color-black: #010204;
+
+:root {
+
+  --color-white: #{$color-white};
+  --color-black: #{$color-black};
+
+  // Primary Colors (normal usage)
+  --color-primary-lighter: #{$color-primary-50};
+  --color-primary-light: #{$color-primary-300};
+  --color-primary: #{$color-primary-400};
+  --color-primary-dark: #{$color-primary-500};
+  --color-primary-darker: #{$color-primary-700};
+
+  --color-primary-text: #{$color-white};
+
+  // Numbered color weights (similar to font weight)
+  --color-primary-50: #{$color-primary-50};
+  --color-primary-100: #{$color-primary-100};
+  --color-primary-200: #{$color-primary-200};
+  --color-primary-300: #{$color-primary-300};
+  --color-primary-400: #{$color-primary-400};
+  --color-primary-500: #{$color-primary-500};
+  --color-primary-600: #{$color-primary-600};
+  --color-primary-700: #{$color-primary-700};
+  --color-primary-800: #{$color-primary-800};
+  --color-primary-900: #{$color-primary-900};
+
+  // Secondary Colors (normal usage)
+  --color-secondary-lighter: #{$color-secondary-50};
+  --color-secondary-light: #{$color-secondary-300};
+  --color-secondary: #{$color-secondary-400};
+  --color-secondary-dark: #{$color-secondary-500};
+  --color-secondary-darker: #{$color-secondary-700};
+
+  --color-secondary-text: #{$color-black};
+
+  // Numbered color weights (similar to font weight)
+  --color-secondary-50: #{$color-secondary-50};
+  --color-secondary-100: #{$color-secondary-100};
+  --color-secondary-200: #{$color-secondary-200};
+  --color-secondary-300: #{$color-secondary-300};
+  --color-secondary-400: #{$color-secondary-400};
+  --color-secondary-500: #{$color-secondary-500};
+  --color-secondary-600: #{$color-secondary-600};
+  --color-secondary-700: #{$color-secondary-700};
+  --color-secondary-800: #{$color-secondary-800};
+  --color-secondary-900: #{$color-secondary-900};
+
+  // Neutrals (normal usage)
+  --neutral-layer-1: #{$color-neutral-50};
```

Comment:
Move to naming of `neutral-layer-1-color` `neutral-layer-1-border`, `neutral-layer-1-text`.

Also adding something for background (with text and border color)